.github/workflows/test_ci.yml

```diff
@@ -0,0 +1,91 @@
+name: Test
+
+on:
+  push:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+          - name: ubuntu_16_04-x86_64
+            os: ubuntu-16.04
+            TARGET_CPU: x86-64
+
+          - name: ubuntu_18_04-x86_64
+            os: ubuntu-18.04
+            TARGET_CPU: x86-64
+
+          - name: ubuntu_20_04-x86_64
+            os: ubuntu-20.04
+            TARGET_CPU: x86-64
+
+          - name: osx_11_0-x86_64
+            os: macos-11.0
+            TARGET_CPU: x86-64
```

Comment:
The nuget package suffix for os x 11 should be macos-x86_64

Reply:
We can use TARGET_CPU=nehalem as before https://github.com/imazen/imageflow/blob/b86389026f84565c17c0f4d4e03e12c83cfd2619/.travis.yml#L22